### PR TITLE
Slightly revamped styling and added content

### PR DESCRIPTION
**Pull request by noraacode**
I’m going to work on “Styling” first.

1. Made the text color all match to red. 
2. Adjusted text font-size to be the same
3. Added a title box to the Six Great Principles and combined both the title and principles as one box —> switched Ridge borders to Groove 
4. Added an English translation in blue text color underneath the Vietnamese counterpart
5. This made the box take the shape of a rectangle rather than a square 

Now, I am going to add content in the content box
1. I made a News box in the top half. 
- The News title has a linear animal from side to side
- There's an image for upcoming celebrations.
- There is an image for extra announcements; in this case, an after-school tutoring program and its time and dates.
2. I made a Weekly schedule box in the bottom half. 
- Had a description of the temple and the master
- Added images and times for the schedule of events: meditation and Dharma discussion.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -7,7 +7,7 @@ body {
 .wrapper {
   display: grid;
   grid-template-columns: 18% 60% 18%;
-  grid-template-rows: 0.05fr 1fr;  
+  grid-template-rows: 0.05fr 0.03fr 1fr;
```
Comment (duongand) [inline styles.css]: Remove this until you actually work on the navigation bar

File: index.html [hunk 2]
```diff
@@ -63,8 +81,57 @@
...
+          Friday Sessions:
+          </br>
+          Meditation Session: 6:00-7:00 PM EST
+          </br>
+          Buddhist Discussion: 7:00-8:00 PM EST
+          </div>
+          <div class="schedule-right">
+          <img
+          src="assets/meditation2.jpg"
+          alt="Thich Giao Dao meditates with guests"
```
Comment (duongand) [inline index.html]: Add "height" property to match photo to the left

File: styles.css [hunk 3]
```diff
@@ -43,28 +43,49 @@ body {
 .left-panel {
   display: flex;
   flex-direction: column;
-  min-height: 100vh;
+  justify-content: space-evenly;
   background-image: url(assets/floral-frost.jpg);
   background-repeat: repeat;
+  color: red;
```
Comment (duongand) [inline styles.css:49]: Add percentage of padding

**Comment by duongand** [comment]
Remove the obsolete < br > elements and replace with CSS styling

**Comment by duongand** [comment]
Remove the images not used for the most recent commits

File: styles.css [hunk 4]
```diff
@@ -43,28 +43,49 @@ body {
...
 
 .master {
   display: flex;
   flex-direction: column;
   font-family: 'Times New Roman', Times, serif; 
   font-weight: bold; 
-  font-size: .9rem; 
-  text-align: center; color: #CC6600;
+  font-size: 1.2rem; 
+  text-align: center; color: red;
```
Comment (duongand) [inline styles.css]: Why are they on the same line?

File: styles.css (hunk 3 above)
Comment (noraacode) [reply styles.css:49]: What? For what

**Review by duongand — APPROVED** [review]
LGTM

**Pull request merged**